src/components/profile/profile-test.stories.tsx

```diff
@@ -34,6 +34,13 @@ DefaultStory.story = {
     name: "John Smith",
     text: "Some other text about John here",
     src: "",
+    m: "",
+    mt: "",
```

Comment:
**issue**: `mt`, `mr`, `mb`, `ml` don't seem to work - when inspecting the styles the margin isn't being set. The rest of the margin props seem to work fine.

Reply:
Yeah, this is weird as all other props do work when they're passed manually to the test component. I'll try fix it when i'm back

Reply:
the last two args are `mx` and `my` they will override any value you pass to `mt`, `mb`, `ml` or `mr`. I think adopting the same approach we have [here](https://github.com/Sage/carbon/blob/fd017ca9e8065c1f1316076f00e27127d324e1ec/src/components/button/button-test.stories.tsx) for `href` will work